code/modules/reagents/reagent_containers.dm

```diff
@@ -156,6 +156,16 @@
 	if(beaker_weakness_bitflag & PH_WEAK)
 		if((reagents.pH < 1.5) || (reagents.pH > 12.5))
 			START_PROCESSING(SSobj, src)
+	else if((reagents.pH < -3) || (reagents.pH > 17))
+		var/list/seen = viewers(5, get_turf(src))
+		var/iconhtml = icon2html(src, seen)
+		for(var/mob/M in seen)
+			to_chat(M, "<span class='notice'>[iconhtml] \The [src]'s is damaged by the super pH and begins to deform!</span>")

```

Comment:
Use visible_message() instead, please. Delete these awful lines.
```
	visible_message("<span class='notice'>[icon2html(src, seen)] \The [src] is damaged by the super pH and begins to deform!</span>")
```

Reply:
I've tried that, but [iconhtml] never works if i use visible_message(), do you know how to fix it?

Reply:
Ok then, try and see if this works instead, icon2html is somewhat stringy.
```
	visible_message("<span class='notice'>[icon2html(src, viewers(src)] \The [src] is damaged by the super pH and begins to deform!</span>")
```